components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/DisplayToolbar.kt

```diff
@@ -356,13 +362,12 @@ internal class DisplayToolbar(
         val navigationActionsWidth = navigationActions
             .asSequence()
             .mapNotNull { it.view }
-            .fold(0) { usedWidth, view ->
-                val viewLeft = usedWidth
+            .fold(0) { _, view ->
```

Comment:
This loop looks like it's supposed to sum together the width of all the actions, so the `usedWidth` argument shouldn't be discarded.

Reply:
Ah, my bad. Fixed!